scripts/build_and_test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Copyright 2020 The ElasticDL Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# Build elasticdl.org/elasticdl Go package
+go env -w GOPROXY=https://goproxy.cn,https://developer.aliyun.com/mirror/goproxy,direct
+cd /elasticdl && \
+make -f elasticdl/Makefile && \
+cp -r /elasticdl/elasticdl/go/ /root/elasticdl && \
```

Comment:
The directory `/root/elasticdl` doesn't exist on the host.  This line prevents this script from being used in on-host development. Let us see if we can fix it in a future PR.